src/InstrumentationEngine.Lib/InstrumentationMethod.cpp

```diff
@@ -83,7 +83,17 @@ HRESULT MicrosoftInstrumentationEngine::CInstrumentationMethod::InitializeCore(
     WCHAR wszModuleFullPath[MAX_PATH];
     memset(wszModuleFullPath, 0, MAX_PATH);
     wcscpy_s(wszModuleFullPath, MAX_PATH, m_bstrModuleFolder);
-    PathCchAppend(wszModuleFullPath, MAX_PATH, m_bstrModule);
+
+    // For Windows 7 support, we cannot use PathCch functions.
+    //PathCchAppend(wszModuleFullPath, MAX_PATH, m_bstrModule);
+    if (!wcslen(wszModuleFullPath) + m_bstrModule.Length() < MAX_PATH - 1 ||
```

Comment:
Did you mean `!`**(**`wcslen(...) + m_bstrModule.Length()`**)**

Reply:
yep or swap `<` with `>`